README.md

```diff
@@ -63,12 +63,28 @@ end
     $ mix escript.install hex protobuf
     ```
 
-3. Generate Elixir code using `protoc`:
+    Then:
+
+    Generate Elixir code using `protoc`:
 
     ```bash
     $ protoc --elixir_out=./lib helloworld.proto
     ```
 
+2b. Add `proto` to your project's compiler list:
+
+  ```elixir
+  def project do
+    [
+      ...
+      compilers: [:proto | Mix.compilers()]
+      ...
+    ]
+  end
+  ```
+
```

Comment:
```suggestion

  Your `.proto` files will be compiled whenever you type `mix compile` or `iex -S mix`.
```

Reply:
Actually this would be a good place to explain to the user the files will only be re-compiled if there is any change to the .proto file, or the config in mix.exs is changed

Reply:
updated README.md